src/dispatch/participant/models.py

```diff
@@ -46,7 +48,10 @@ class ParticipantBase(DispatchBase):
 
 
 class ParticipantCreate(ParticipantBase):
-    participant_role: Optional[List[ParticipantRoleCreate]] = []
+    participant_roles: Optional[List[ParticipantRoleCreate]] = []
```

Comment:
This will affect some frontend UI elements, also we should modify `ParticipantRead` as well.